src/main/java/com/amazon/dlic/auth/http/saml/HTTPSamlAuthenticator.java

```diff
@@ -397,7 +409,7 @@ String buildLogoutUrl(AuthCredentials authCredentials) {
 
     }
 
-    private void initLogoutUrl(RestRequest restRequest, ThreadContext threadContext, AuthCredentials authCredentials) {
+    private void initLogoutUrl(ThreadContext threadContext, AuthCredentials authCredentials) {
```

Comment:
Why don't we need the request parameter here?

Reply:
We never used it, there are a couple of places where I found unused parameters and removed them rather than refactoring RestRequest -> SecurityRequest